libkbfs/dirty_bcache.go

```diff
@@ -357,7 +337,7 @@ func (d *DirtyBlockCacheStandard) maybeDecreaseBuffer(start time.Time,
 		if d.syncBufferCap < d.minSyncBufCap {
 			d.syncBufferCap = d.minSyncBufCap
 		}
-		d.logLocked("Writes blocked for %s (%f%% of timeout), "+
+		d.log.Debug("Writes blocked for %s (%f%% of timeout), "+
```

Comment:
I'm sad to lose my TODO above about passing in contexts here.  What about `CDebugf(context.TODO(),...` here and below instead?